Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -76,10 +80,27 @@
                     {
                         textValue: texts.notActive,
                         remoteValue: false
-                    }]
+                    }];
             }
 
-            //Build and launch kendo grid
+
+            var self = this;
+            var showInactiveSystems = ItSystem.Settings.OverviewState.getShowInactiveSystems($window, user.id, pageName);
+
+            function getToggleActiveSystemsFilterBtnText(): string {
+                return showInactiveSystems
+                    ? "Vis aktive IT Systemer"
+                    : "Vis \"ikke aktive\" IT Systemer";
+            }
+                                              
+            function toggleActiveSystemsMasterFilter(): void {
+                showInactiveSystems = !showInactiveSystems;
+                ItSystem.Settings.OverviewState.setShowInactiveSystems($window, user.id, pageName, showInactiveSystems);
+                getToggleActiveSystemsFilterBtnText();
```

Comment:
why are you calling that function?... it has no effect but just returns a string